lib/src/boom_menu.dart

```diff
@@ -17,11 +17,17 @@ class BoomMenu extends StatefulWidget {
   final Color foregroundColor;
   final double elevation;
   final ShapeBorder fabMenuBorder;
+  final Alignment fabAlignment;
 
   final double marginLeft;
   final double marginRight;
   final double marginBottom;
 
+  final double fabPaddingLeft;
+  final double fabPaddingRight;
+  final double fabPaddingTop;
+  final double fabPaddingBottom;
+
```

Comment:
```suggestion
final double fabPaddingTop;
```
Remove this Line(28). it's make a Bottom overflowed error.